troubleshooting/rocksdb/src/solution/java/com/ververica/flink/training/solutions/RocksDBTuningJobOptionsFactory.java

```diff
@@ -0,0 +1,133 @@
+package com.ververica.flink.training.solutions;
+
+import org.apache.flink.configuration.ConfigOption;
+import org.apache.flink.configuration.ReadableConfig;
+import org.apache.flink.configuration.description.Description;
+import org.apache.flink.contrib.streaming.state.ConfigurableRocksDBOptionsFactory;
+import org.apache.flink.contrib.streaming.state.DefaultConfigurableOptionsFactory;
+
+import org.rocksdb.ColumnFamilyOptions;
+import org.rocksdb.CompressionType;
+import org.rocksdb.DBOptions;
+import org.rocksdb.InfoLogLevel;
+import org.rocksdb.Logger;
+import org.slf4j.LoggerFactory;
+
+import java.time.Duration;
+import java.util.Collection;
+
+import static org.apache.flink.configuration.ConfigOptions.key;
+import static org.apache.flink.configuration.description.LinkElement.link;
+
+/**
+ * Extended RocksDB option factory which enables further settings from
+ * https://github.com/facebook/rocksdb/wiki/RocksDB-Tuning-Guide.
+ */
+@SuppressWarnings("unused")
+public class RocksDBTuningJobOptionsFactory implements ConfigurableRocksDBOptionsFactory {
+    public static final org.slf4j.Logger LOG =
+            LoggerFactory.getLogger(RocksDBTuningJobOptionsFactory.class);
+
+    private static final long serialVersionUID = 1L;
+
+    private static final ConfigOption<Boolean> JAVA_LOGGING =
+            key("state.backend.rocksdb.custom.javalogging")
+                    .booleanType()
+                    .defaultValue(true)
+                    .withDescription("Redirect RocksDB logging to Java (max reduce performance)");
+
+    private static final ConfigOption<Duration> DUMP_STATS_INTERVAL =
+            key("state.backend.rocksdb.custom.dumpstats")
+                    .durationType()
+                    .defaultValue(Duration.ofMinutes(5))
+                    .withDescription("If not zero, dump RocksDB stats this often (in seconds)");
```

Comment:
It's a duration, I'm assuming we just need to pass the value to RockDB in seconds. 
```suggestion
                    .withDescription("If not zero, dump RocksDB stats this often");
```